cypress/integration/channel/message_draft_with_attachment_then_switch_channel_spec.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+function createNewChannel(name, header, purpose, type) {
+    const createChannel = '#create' + type + 'Channel';
+    const createChannelName = '#newChannelName';
+    const createChannelPurpose = '#newChannelPurpose';
+    const createChannelHeader = '#newChannelHeader';
+
+    cy.get(createChannel).click();
+    cy.get(createChannelName).type(name);
+    cy.get(createChannelPurpose).type(purpose);
+    cy.get(createChannelHeader).type(header);
+
+    cy.contains('Create New Channel').click({force: true});
+    if (cy.get('#createChannelError')) {
```

Comment:
This line failed when no error in creating a channel.  

I really like this function to drive UI in creating a channel.  However, since we're not testing such channel creation flow, I'd lean towards doing that via API.  We have `cy.apiCreateChannel` that you may use.

Reply:
+1 to using API for channel creation usage.